source/educators/navigation/advanced_features.rst

```diff
@@ -0,0 +1,95 @@
+:orphan:
+
+Advanced Features
+###################
+
+.. _Offer Different Content to Different Groups:
+
+Offer Different Content to Different Groups (Cohorts)
+*******************************************************
+
+.. toctree::
+   :maxdepth: 2
```

Comment:
I wonder if having maxdepth 2 on these pages makes the pages a bit more unwieldly.

https://docsopenedxorg--601.org.readthedocs.build/en/601/educators/navigation/advanced_features.html

Reply:
I'm also a little torn on that. Here is my argument for keeping it at 2: 

1. So far, we've broken these docs outs into their diataxis type but we still often have several how-to's in a single file  (e.g. Enable a Survey, Add a Survey, Edit a Survey) when they go together, and so we don't have 1,000+ files. If we did TOC level 1, we'd show something like this:
<img width="733" alt="image" src="https://github.com/user-attachments/assets/ff8f9697-cd64-4d3c-a589-41c1d33e03c7">

Instead of what we have now:
<img width="571" alt="image" src="https://github.com/user-attachments/assets/ce0f016a-7e52-47e6-b2e7-1fce0247d593">

OK, actually that's not bad. But I think it would involve renaming all the level 1 docs to make sure they action is clear (e.g. "Manage Survey Tool" instead of "Survey Tool"). This is something I'm open to do at the end of the project with leftover hours (I think there will be some). 

2. It makes the right sidebar navigation often redundant with the center panel. Especially for features where we only have one doc. e.g: 
<img width="434" alt="image" src="https://github.com/user-attachments/assets/a2db36db-193c-47dd-967c-f11f01e73877">

I say we bring this to the FC Group in our next meeting. But push this with 2 levels for now.

Reply:
OK, I agree with you. Leave at 2 levels for now, reasses after this merges.